test/stack_controller_test.go

```diff
@@ -183,19 +184,29 @@ var _ = Describe("Stack Controller", func() {
 		}
 	})
 
+	var toDelete []string
+
+	defer func() {
+		for _, d := range toDelete {
+			_ = os.RemoveAll(d)
+		}
+	}()
+
 	// Tip: avoid adding tests for vanilla CRUD operations because they would
 	// test the Kubernetes API server, which isn't the goal here.
 
 	It("Should deploy a simple stack locally successfully", func() {
 		var stack *pulumiv1alpha1.Stack
 		// Use a local backend for this test.
 		// Local backend doesn't allow setting slashes in stack name.
-		stackName := fmt.Sprintf("%s-local-simple-stack-dev-%s", stackOrg, randString())
+		const stackName = "dev"
 		fmt.Fprintf(GinkgoWriter, "Stack.Name: %s\n", stackName)
 
 		// Tests run from outside the container. Using the safest existing directory to store
 		// local state for the sake of this test.
-		const backendDir = "/tmp"
+		backendDir, err := ioutil.TempDir("", "local-state")
+		Ω(err).ShouldNot(HaveOccurred())
```

Comment:
Was the omega symbol intended?

Reply:
Its part of the gomega library the test framework uses: https://onsi.github.io/gomega/#making-assertions